File-Formats/XlsIO/Create-read-edit-excel-files-in-maui-c-sharp.md

```diff
@@ -242,85 +242,96 @@ using (ExcelEngine excelEngine = new ExcelEngine())
     MemoryStream ms = new MemoryStream();
     workbook.SaveAs(ms);
     ms.Position = 0;
-	
+
     //Saves the memory stream as a file.
-    DependencyService.Get<ISave>().SaveAndView("Invoice.xlsx", "application/excel", ms);
+    SaveService saveService = new SaveService();
+    saveService.SaveAndView("Output.xlsx", "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet", ms);
 }
 {% endhighlight %}
 {% endtabs %}  
 
+## Save Service class in portable project
+
+Add the new class file with name as **SaveService** to the **Project** and add below code in it.
```

Comment:
Please mention the usage of this class.

Reply:
I have mentioned the usage as suggested.